tools/spelling/spelling_dictionary.txt

```diff
@@ -1379,6 +1379,7 @@ subtrees
 subtype
 subtypes
 subzone
+successfull
```

Comment:
this is not the correct spelling. Remove the last L.

https://www.dictionary.com/browse/successful

You can drop this change and fix the usage.